### PR TITLE
Feature: Add vue package

### PR DESCRIPTION
**Pull request by wonderbeel**
### Description
Continuation of https://github.com/blocknative/web3-onboard/pull/911

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [ ] This PR passes the Circle CI checks

### CODE REVIEW

File: packages/vue/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@web3-onboard/vue",
+  "version": "2.2.8-alpha.1",
```
Comment (Adamj1232) [inline packages/vue/package.json:3]:
```suggestion
  "version": "2.0.0-alpha.1",
```

File: packages/vue/README.md [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+    const { lastConnectedTimestamp } = useOnboard()
+    return { lastConnectedTimestamp }
+  }
+}
+</script>
+
+<template>
+  <span>Your last connection timestamp was: {{ lastConnectedTimestamp }}</span>
+</template>
+```
```
Comment (Adamj1232) [inline packages/vue/README.md:364]: Are there any build/bundler challenges you ran into when using the Vue package?

Comment (wonderbeel) [reply packages/vue/README.md:364]: Nothing in particular Vue related, had some trouble with vite but it is already known (also how to workaround with polyfills)

File: packages/vue/package.json (hunk 1 above)
Comment (taylorjdawson) [reply packages/vue/package.json:3]: @wonderbeel can you update the version?

Comment (wonderbeel) [reply packages/vue/package.json]: Done

**Review by taylorjdawson — APPROVED** [review]

**Comment by taylorjdawson** [comment]
@wonderbeel looks good!

**Pull request merged**